api_tests/subscriptions/views/test_subscriptions_detail.py

```diff
@@ -19,7 +21,12 @@ def user_no_auth(self):
 
     @pytest.fixture()
     def notification(self, user):
-        return NotificationSubscriptionFactory(user=user)
+        return NotificationSubscriptionFactory(
+            notification_type=NotificationType.Type.USER_FILE_UPDATED.instance,
```

Comment:
These tests weren't quite right because they didn't use the subscribed object attribute, only type and user.